vhdbuilder/packer/install-dependencies.sh

```diff
@@ -370,23 +370,23 @@ K8S_VERSIONS="
 1.16.4
 1.16.1
 1.14.6_v0.0.5
-1.14.7-hotfix.20200326
```

Comment:
Need to keep these old image until we upgraded all ccp chart to use the  new images